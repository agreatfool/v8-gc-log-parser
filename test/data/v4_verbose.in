[7144:0x102004a00] Memory allocator,   used: 111556 KB, available: 1387580 KB
[7144:0x102004a00] New space,          used:   1497 KB, available:  14626 KB, committed:  32248 KB
[7144:0x102004a00] Old space,          used:  60641 KB, available:   9666 KB, committed:  70398 KB
[7144:0x102004a00] Code space,         used:   1767 KB, available:    205 KB, committed:   2266 KB
[7144:0x102004a00] Map space,          used:    273 KB, available:    777 KB, committed:   1070 KB
[7144:0x102004a00] Large object space, used:      0 KB, available: 1386539 KB, committed:      0 KB
[7144:0x102004a00] All spaces,         used:  64179 KB, available: 1411816 KB, committed: 105982 KB
[7144:0x102004a00] External memory reported:     24 KB
[7144:0x102004a00] Total time spent in GC  : 64.9 ms
[7144:0x102004a00] Memory allocator,   used: 112580 KB, available: 1386556 KB
[7144:0x102004a00] New space,          used:    203 KB, available:  15920 KB, committed:  32248 KB
[7144:0x102004a00] Old space,          used:  60652 KB, available:  10012 KB, committed:  71406 KB
[7144:0x102004a00] Code space,         used:   1672 KB, available:    564 KB, committed:   2266 KB
[7144:0x102004a00] Map space,          used:    238 KB, available:    812 KB, committed:   1070 KB
[7144:0x102004a00] Large object space, used:      0 KB, available: 1385515 KB, committed:      0 KB
[7144:0x102004a00] All spaces,         used:  62767 KB, available: 1412825 KB, committed: 106990 KB
[7144:0x102004a00] External memory reported:     16 KB
[7144:0x102004a00] Total time spent in GC  : 68.1 ms