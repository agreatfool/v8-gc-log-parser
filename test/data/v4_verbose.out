[
  {
    "pid": 7144,
    "isolate": "0x102004a00",
    "type": "verbose",
    "data": {
      "memory_allocator": {
        "used": 111556,
        "available": 1387580
      },
      "new_space": {
        "used": 1497,
        "available": 14626,
        "committed": 32248
      },
      "old_space": {
        "used": 60641,
        "available": 9666,
        "committed": 70398
      },
      "code_space": {
        "used": 1767,
        "available": 205,
        "committed": 2266
      },
      "map_space": {
        "used": 273,
        "available": 777,
        "committed": 1070
      },
      "large_object_space": {
        "used": 0,
        "available": 1386539,
        "committed": 0
      },
      "all_spaces": {
        "used": 64179,
        "available": 1411816,
        "committed": 105982
      },
      "external_memory_reported": 24,
      "total_time_spent_in_gc": 64.9
    }
  },
  {
    "pid": 7144,
    "isolate": "0x102004a00",
    "type": "verbose",
    "data": {
      "memory_allocator": {
        "used": 112580,
        "available": 1386556
      },
      "new_space": {
        "used": 203,
        "available": 15920,
        "committed": 32248
      },
      "old_space": {
        "used": 60652,
        "available": 10012,
        "committed": 71406
      },
      "code_space": {
        "used": 1672,
        "available": 564,
        "committed": 2266
      },
      "map_space": {
        "used": 238,
        "available": 812,
        "committed": 1070
      },
      "large_object_space": {
        "used": 0,
        "available": 1385515,
        "committed": 0
      },
      "all_spaces": {
        "used": 62767,
        "available": 1412825,
        "committed": 106990
      },
      "external_memory_reported": 16,
      "total_time_spent_in_gc": 68.1
    }
  }
]